flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/EmbeddedRocksDBStateBackend.java

```diff
@@ -91,12 +91,6 @@
 public class EmbeddedRocksDBStateBackend extends AbstractManagedMemoryStateBackend
         implements ConfigurableStateBackend {
 
-    /** The options to chose for the type of priority queue state. */
-    public enum PriorityQueueStateType {
```

Comment:
not sure about this change, it could break existing pipelines because it is exposed via `setPriorityQueueStateType`

Reply:
You mean breaking because the location of the enum changes? I can also leave the enum here.

Reply:
yes, let's leave the enum there for now. The class is marked as `PublicEvolving` but this is actually not correct. The RocksDB State Backend has evolved to `Public` nowadays, so we need to be careful.